src/pages/Completed/Completed.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+import Button from '../../components/Button/Button';
+
+function Completed(props) {
```

Comment:
props를 받지 않는다면 아예 지워주세요!
```suggestion
function Completed() {
```